packages/superset-ui-query/src/processMetrics.ts

```diff
@@ -1,25 +1,11 @@
-import { QueryFormData } from './types/QueryFormData';
+import { QueryFormDataMetric } from './types/QueryFormData';
 import { QueryObjectMetric } from './types/Query';
-import { MetricKey } from './types/Metric';
 import convertMetric from './convertMetric';
 
-export default function processMetrics(formData: QueryFormData) {
-  // Use Array to maintain insertion order
-  // for metrics that are order sensitive
+export default function processMetrics(rawMetrics: QueryFormDataMetric[]) {
   const metrics: QueryObjectMetric[] = [];
```

Comment:
```ts
return rawMetrics.map(convertMetric);
```

Reply:
need flatmap

Reply:
oh wait I was looking at the wrong side of the diff 🤦‍♂️